doc/tutorials/partition-mode.rst

```diff
@@ -0,0 +1,332 @@
+.. _partition_mode:
+
+Using partition mode
+#########################
+This tutorial describes all the steps you need to take in order to be able to
+use partition mode with the ACRN hypervisor. Partition mode provides an
+isolation of PCI devices to a given UOS so that the PCI devices can be used
+exclusively by that UOS, bypassing the ACRN hypervisor and achieving near-native
+performance.
+
```

Comment:
I understand there is no Service OS anymore when using this mode, is that correct? If true, we should mention this to make sure people understand what this new mode is about. I'm actually thinking that this deserves to be explained somewhere in our [introduction](https://projectacrn.github.io/latest/introduction/index.html#introduction-to-project-acrn) too.

Reply:
agree, we should have a separate doc to introduce partition mode.  what it is ,and why partition, then have a documentation to describe how to enable and config this mode, because default is sharing mode